wiki/Tournaments/OWC/2/en.md

```diff
@@ -238,79 +269,334 @@ This competition has come to an end and resulted in the following podium:
+| C | ![][flag_ID] Indonesia | 1 | **3** | ![][flag_UA] **Ukraine** | [#1](https://osu.ppy.sh/community/matches/2023842) |
+| G | ![][flag_BR] **Brazil** | **3** | 0 | ![][flag_BG] Bulgaria | [#1](https://osu.ppy.sh/community/matches/2026764) |
+| A | ![][flag_DK] Denmark | 0 | **3** | ![][flag_DE] **Germany** | [#1](https://osu.ppy.sh/community/matches/2026370) |
+
+Sunday, 20 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| E | ![][flag_CL] Chile | 1 | **3** | ![][flag_FR] **France** | [#1](https://osu.ppy.sh/community/matches/2027339) |
+| H | ![][flag_TH] Thailand | 0 | **3** | ![][flag_US] **United States** | [#1](https://osu.ppy.sh/community/matches/2028163) |
+| B | ![][flag_HU] **Hungary** | **3** | 1 | ![][flag_NZ] New Zealand | [#1](https://osu.ppy.sh/community/matches/2030888) |
+| D | ![][flag_HK] **Hong Kong** | **3** | 0 | ![][flag_VN] Vietnam | [#1](https://osu.ppy.sh/community/matches/2031284) |
+| F | ![][flag_UY] Uruguay | 2 | **3** | ![][flag_MY] **Malaysia** | [#1](https://osu.ppy.sh/community/matches/2033451) |
+
+Monday, 21 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| B | ![][flag_AR] **Argentina** | **3** | 2 | ![][flag_AU] Austria | [#1](https://osu.ppy.sh/community/matches/2034560) |
+
+Friday, 25 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| C | ![][flag_NO] Norway | 2 | **3** | ![][flag_PL] **Poland** | [#1](https://osu.ppy.sh/community/matches/2056490) |
+
+Saturday, 26 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| G | ![][flag_SE] Sweden | 0 | **3** | ![][flag_TW] **Taiwan** | [#1](https://osu.ppy.sh/community/matches/2067611) |
+| E | ![][flag_LV] Latvia | 0 | **3** | ![][flag_PT] **Portugal** | [#1](https://osu.ppy.sh/community/matches/2070120) |
+
+Sunday, 27 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| D | ![][flag_CA] Canada | 1 | **3** | ![][flag_KR] **South Korea** | [#1](https://osu.ppy.sh/community/matches/2072809) |
+
 ## Ruleset
 
 ### Tournament rules
 
-1. No more team than 1 from 1 country can sign-up to the tournament.
-2. All of the team players' flag in the profile must be the same as their country's. However, it can be got around in special cases.
-3. Any form of unfair behaviour (such as cheating, insulting other players, racism) will be punished with an immediate banishment of the player/the team from the tournament.
+1. The osu! World Cup is a 4v4, country-based team tournament played on the osu! game mode.
+2. Beatmap scoring is based on **[ScoreV1](/wiki/Score#scorev1)**.
+3. The beatmaps for each round will be announced by the tournament host on the discussion thread in advance one before the actual matches take place. Only these beatmaps will be used during the respective matches. 
```

Comment:
> *will be announced by the tournament host on the discussion thread **in advance one before** the actual matches take place*

one *week*? one *day*?

Reply:
oh i forgot to delete the "one" lol

iirc from what i get on the thread the scheduling for this owc (esp. on the later stages) were quite off the chart because of delays and other issues. this ultimately affected the mappool announcement time as well, hence why i simply wanted to write ``will be announced by the tournament host on the discussion thread in advance before the actual matches take place`` without any specific mention about its time

![image](https://user-images.githubusercontent.com/36564236/105468207-50e07000-5cc9-11eb-8a7c-71afd9855bfa.png)